### PR TITLE
Milestone3: Creating a Contact form

### PR DESCRIPTION
**Pull request by Cavein254**
This milestone accomplishes the following:

   - completes the mobile portfolio version by adding a contact form
   - Adds the required HTML and CSS styles
   - Ensures there are no linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
...
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <div class="contact">
```
Comment (souad988) [inline index.html]: Good job on adding this section , but this should be a form all inputs and the button should be wrapped in a form and the form needs a method which is POST here and an action attribute where we specify the api to which we send data after submission 

also there are some small styling issues that we need to fix , first is the background color of this section make sure to use the same as in figma , second the button it needs the interaction effects , and last is textareas text and background colors 

temp2 figma footer
temp2 student footer

File: index.html [hunk 2]
```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
...
+          <div class="contact-input">
+            <input type="text" placeholder="Full Name" name="name" required />
+          </div>
```
Comment (souad988) [inline index.html]: well done by making this input required and specifying it's type to text 👍, but we don't need to wrap every single item with a div as it is meant to wrap a block of items you can directly style the inputs , and we need here to specify maxlength attribute  `maxlength="30"`

File: index.html [hunk 3]
```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
...
+          <div class="contact-input">
+            <input
+              type="email"
+              placeholder="Email Address"
+              name="email"
+              required
+            />
+          </div>
+          <div class="info">
+            <textarea placeholder="Write something ..." rows="6"></textarea>
```
Comment (souad988) [inline index.html]: same here `maxlength="500"` and in the requirement this filed needs to be required

File: index.html [hunk 4]
```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
...
+          </div>
+        </div>
+        <div class="contact-btn">
+          <div class="story-btn">
+            <span><a href="#">Get in touch</a></span>
+          </div>
```
Comment (souad988) [inline index.html]: it's better to use button tag as a tag is to redirect user to a new page or bloc in the page but here we have a form and the button is needed to submit data in the form so we can benefit from the type= submit property of the button so when clicked all data in inputs inside the form will be submited to the destination in the action of our form

File: index.html [hunk 5]
```diff
@@ -57,6 +57,313 @@ <h2>I'm a Software Developer</h2>
...
+        <li>
+          <a href="#"><img src="./pics/twitter.svg" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/medium.svg" alt="medium" /></a>
+        </li>
+      </ul>
+      <hr />
+    </div>
+  </footer>
```
Comment (souad988) [inline index.html]: Please make sure to use the same colors in the template the background of this footer is different 
temp2 figma footer
temp2 student footer

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Cavein254 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Cavein254) [reply index.html]: Done :heavy_check_mark:

File: index.html (hunk 3 above)
Comment (Cavein254) [reply index.html]: done :heavy_check_mark:

File: index.html (hunk 4 above)
Comment (Cavein254) [reply index.html]: done :heavy_check_mark:

File: index.html (hunk 5 above)
Comment (Cavein254) [reply index.html]: done :heavy_check_mark:

File: index.html [hunk 6]
```diff
@@ -57,6 +57,329 @@ <h2>I'm a Software Developer</h2>
...
```
Comment (ABDULALI3468) [inline index.html]: As requested by the previous CR please change the background of the footer section to white and remember only the background color is applied to the `textarea element` with the value of the following `background: #FBF8F7;`
`

**Review by Gambit142 — COMMENTED** [review]
Hi @Cavein254 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- This is a pair-programming exercise and as such, you ought to collaborate with your partner. In other words, your partner should have some commits in this milestone. Kindly collaborate with your partner. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 7]
```diff
@@ -57,6 +57,329 @@ <h2>I'm a Software Developer</h2>
...
+              required
+              maxlength="500"
+            ></textarea>
+          </div>
+        </form>
+        <div class="contact-btn">
+          <button class="story-contact-btn" type="submit">
+            <a href="#">Get in touch</a>
+          </button>
+        </div>
```
Comment (ABDULALI3468) [inline index.html]: Please consider also wrapping this submit button in the form element as requested b the previous code reviewer to make it align with the project requirements.

> Moreover, consider using the same `hover and active styling` as used in Figma.

image

File: index.html (hunk 6 above)
Comment (ABDULALI3468) [inline index.html]: Please consider changing the color of icons to `background: #505F79;` as used in the Figma to meet the project requirements.

## Figma Design:-
image

## Your Design:-
image

File: index.html [hunk 8]
```diff
@@ -57,6 +57,329 @@ <h2>I'm a Software Developer</h2>
...
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <form
+          class="contact"
+          action="https://formspree.io/f/xzbqggzq"
+          method="POST"
+        >
+          <div class="contact-input">
```
Comment (ABDULALI3468) [inline index.html:328]: On your page, there is a small gap on the right side of the screen consider fixing it as it has a bad impact on the user experience. Remember your design needs to meet the minimum width of 375px.

## Your Design:-

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Cavein254 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (Cavein254) [reply index.html]: done :heavy_check_mark:

File: index.html (hunk 7 above)
Comment (Cavein254) [reply index.html]: done :heavy_check_mark:

File: index.html (hunk 6 above)
Comment (Cavein254) [reply index.html]: done  :heavy_check_mark:

File: index.html (hunk 8 above)
Comment (Cavein254) [reply index.html:328]: done :heavy_check_mark:

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Cavein254, 

You have done a lot of effort. Your project is complete! Congratulations! 🎉 
But there are some suggestions that you couldn't incorporate correctly  or misunderstood it, as described by previous reviewers. Kindly try to incorporate them.

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions
1. You have mistakenly added background color to whole div class `.interest`. The background color should only be applied to <textarea>. The Full Name and Email Address fields also do not have background as per FIGMA Design.
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**